tar/generate_tar.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# locate config file
+CONFIG_FILE="config-example.yaml"
+
+if [ ! -e "config/$CONFIG_FILE" ]
+then 
+    echo "missing required config file: $CONFIG_FILE"
+    exit 1
+fi
+
+# move the binary back to the root directory
+cp config/$CONFIG_FILE tar/
+cp bin/$OTELCOL_BINARY tar/
+
+# compress the binary and the config into a .tar file
+cd tar && tar -cvzf gcp-otel.tar.gz $OTELCOL_BINARY $CONFIG_FILE
```

Comment:
For consistency, can you move the tarball into the `dist` folder instead

Reply:
tarball is now generated in the dist folder